openvm/src/extraction_utils.rs

```diff
@@ -574,6 +574,7 @@ mod tests {
             base_config,
             vec![],
             crate::PrecompileImplementation::SingleRowChip,
+            DEFAULT_APP_LOG_BLOWUP * 2 + 1,
```

Comment:
Can you explain this?

Reply:
Could it use `DEFAULT_OPENVM_DEGREE_BOUND`?

Reply:
Changed too